docs/ScoreCard.md

```diff
@@ -0,0 +1,55 @@
+# ScoreCard
+Card component that calls attention to particular values.
+
+<div style="align-items: center; text-align: center; width: 100%; display: flex; justify-content: space-evenly; margin-bottom: 20px">
+    <img width="40%" alt="Score Card with multiple highlighted values" src="./images/scoreCard.png"><br/>
+    <img width="40%" alt="Score Card with single badge-style value" src="./images/scoreCard_alt.png"><br/>
+</div>
+
+## Usage
+
+<div style="align-items: center; text-align: center; width: 100%; display: flex; justify-content: space-evenly; margin-bottom: 20px">
+    <img width="80%" alt="Score Card with labels on different parts" src="./images/scoreCardAnatomy.png"><br/>
+</div>
+
+```typescript
+// app.module.ts
+import { ScoreCardModule } from '@pxblue/angular-components';
+...
+imports: [
+    ScoreCardModule
+  ],
+...
+```
+
+```typescript
+// Example with basic body content and header action items
+<pxb-scorecard [headerTitle]="'Title'" >
```

Comment:
Can we show an example that uses badge like react since that's the most complicated and not terribly intuitive

Reply:
Yep updated with a more complicated example; also they should be using the storybook story-source as an alternative source for complicated examples.